src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  margin-bottom: 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  overflow: hidden;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
```

Comment:
Is it a separated block? Remove its styles to separeted file as well

Reply:
I added _header.scss by accident. I didn't change it, as stated in the task "- Do not modify header"

Reply:
it`ll be better to move each bem block to separate file